types/testingutils/ssv_msgs.go

```diff
@@ -52,13 +51,6 @@ var TestAggregatorConsensusData = &types.ConsensusData{
 }
 var TestAggregatorConsensusDataByts, _ = TestAggregatorConsensusData.Encode()
 
-var TestProposerConsensusData = &types.ConsensusData{
```

Comment:
how come we don't use all the code we deleted here? I thought we are not deprecating Bellatrix.